docs/source/install.rst

```diff
@@ -39,22 +39,23 @@ On other platforms, you can get pandoc from
 Installing TeX
 --------------
 
-For converting to PDF, nbconvert uses the TeX document preparation 
-ecosystem. It produces an intermediate ``.tex`` file which is 
-compiled by the XeTeX engine with the LaTeX2e format (via the 
-``xelatex`` command) to produce PDF output. 
+For converting to PDF, nbconvert can either use `Puppeteer <https://pptr.dev>`_
```

Comment:
This section should probably not even mentionthis PR's contant, and certainly not `puppeteer`, as it is not used.

Reply:
I didn't touch those docs in my pass, I can clean it up to reference pyppeteer everywhere that's relevant instead of puppeteer.